lessons/Algebra/surface-area-rect-prism/langs/en-us/pages/sa-rubric.adoc

```diff
@@ -0,0 +1,45 @@
+[.landscape]
+= Rubric: Surface Area of a Rectangular Prism
+++++
+<style>
+	td { font-size: 9pt !important; }
+	td:first-of-type { font-weight: bold !important; }
+</style>
+++++
+
+Important: Students, be sure to sumbit all components of this project, including (1) your published Pyret program link, (2) the rectangular prism that you built, and (3) Surface Area of a Prism - Practice.
+
+
+@vspace{1ex}
+
+[cols="2,4,4,4", options="header"]
+|===
+|
+| Wow!
+| Getting There
+| Needs Improvement
+
+
+| My Program Submission
+| Running my code produces no errors. In Part 1, I correctly defined all six faces of a rectangular prism. In Part 2, I correctly coded the list of faces. In Part 3, I wrote definitions that calculate the area of each face. In Part 4, I wrote code that correctly computes surface area.
+| Running my code produces no errors. There is a small error in one or two of the sections of my Definitions Area.
+| Running my code produces errors. There area errors in three or more of the sections of my Definitions Area.
+
+
+| My Rectangular Prism
+| The rectangular prism that I constructed looks like a rectangular prism! On each face, I've written its area with correct units.
+| The rectangular prism that I constructed looks a lot like a rectangular prism. On each face, I've written its area (but I may have made one or two mistakes).
+| I failed to cut out the faces of my rectangular prism and tape them together / the faces of my rectangular prism do not fit together because of errors in my program.
+
+
+| Surface Area Practice
+
+| I've completed each of the practice problems, being sure to show my work and provide correct units. I gave an incorrect response for no more than one problem.
```

Comment:
I _think_ you mean "more than one way" problem. But I'm not 100% on that.

Reply:
Flannery also created a practice problems page. That is what I'm referring to.